examples/braket_features/Native_gates_on_IonQ.ipynb

```diff
@@ -0,0 +1,1666 @@
+{
```

Comment:
<p>Why do we need this function? It seems like we're adding a fairly complex function just to make the graph below less complex?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
While not 100% necessary, it was really convenient to use this function to compare about the usual QFT algorithms. It  tough to visually inspect the plots for high qubit counts to confirm they are correct, but this function made it easier for me.

Reply:
Can you add some text above it explaining what it is and why you're adding it?